motionplan/motionPlanner_test.go

```diff
@@ -7,19 +7,20 @@ import (
 	"testing"
 
 	"github.com/golang/geo/r3"
+	"github.com/viamrobotics/visualization"
 	"go.uber.org/zap"
 	commonpb "go.viam.com/api/common/v1"
 	motionpb "go.viam.com/api/service/motion/v1"
 	"go.viam.com/test"
 
-	frame "go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/referenceframe"
```

Comment:
There are a lot of line changes in this file as a result of this change.  Does anyone have thoughts about slowly trying to move away from using this shorthand in files?  It's super confusing that some files use `frame` and others `referenceframe`.  I think we should be consistent here

Reply:
Broadly speaking yes we should move away from the shorthand.

However; at the moment we have a bunch of different parallel work going on that this will affect, so let's not put it in this PR, and instead make a ticket. Otherwise this will cause a *ton* of merge conflicts where there doesn't need to be any.

Reply:
Sounds good will revert